grpc/src/client/load_balancing/child_manager.rs

```diff
@@ -77,12 +89,18 @@ pub trait ResolverUpdateSharder<T: ChildIdentifier>: Send {
 impl<T: ChildIdentifier> ChildManager<T> {
     /// Creates a new ChildManager LB policy.  shard_update is called whenever a
     /// resolver_update operation occurs.
-    pub fn new(update_sharder: Box<dyn ResolverUpdateSharder<T>>) -> Self {
-        Self {
+    pub fn new(
+        update_sharder: Box<dyn ResolverUpdateSharder<T>>
+    ) -> Self {
+        ChildManager {
```

Comment:
```suggestion
        Self {
```

I think is preferred, as it was before.